app/controllers/admin/auctions_controller.rb

```diff
@@ -29,8 +29,8 @@ def create
       flash[:success] = I18n.t('controllers.admin.auctions.create.success')
       redirect_to admin_auctions_path
     else
-      flash[:error] = auction.errors.full_messages.to_sentence
-      @view_model = Admin::NewAuctionViewModel.new
+      flash.now[:error] = auction.errors.full_messages.to_sentence
+      @view_model = Admin::NewAuctionViewModel.new(auction)
```

Comment:
yes! also all other fields. noticed when testing this manually that we were not preserving fields values when there was an error, which is annoying for admins